src/model/model.rs

```diff
@@ -754,6 +795,69 @@ mod test {
         assert!(!rule.check(Domain("www.actcast.io".to_owned(), 32768), Udp));
     }
 
+    #[test]
+    fn domain_wildcard() {
+        use Address::Domain;
+        use RulePattern::*;
+        struct Case {
+            wildcard: String,
+            valid_domains: Vec<String>,
+            invalid_domains: Vec<String>,
```

Comment:
Can you change these names from valid/invalid_domains? (e.g. match/not_match_~)
`example.com` is not satisfy pattern `*.*.example.com` but **not** an invalid domain.

Reply:
done with 5bc54c0